README.md

```diff
@@ -2,4 +2,53 @@
 
 Northeastern Electric Racing Project Management Dashboard
 
-All issues and new features are handled in the Github Repository Issues tab
\ No newline at end of file
+---
+### Contributor guide:
+- First time project installation:
+	- Decide which folder you would like to place the NER-PM-Dashboard folder in
+	- Open your Command Line Interface tool of choice in the folder (e.g. Terminal for Mac, Command Prompt for Windows)
+	- Use the `git clone [repo link]` command as such: `git clone https://github.com/Northeastern-Electric-Racing/NER-PM-Dashboard.git`
+	- Now navigate to the directory by typing `cd NER-PM-Dashboard`
+
+- To create an issue (or suggest a feature):
+	- Open the github URL online
+	- Go to the issues tab and click add new issue
+	- Give it a name that gives the reader an idea of whats going on. Names should follow this format: `[Page or "General"] - [Description]`. Save all detailed info for the description
+	- The naming convention is as follows: `[page on site changed] - [name of change]`. (e.g. Gantt - Restyle & Move the "View Full Gantt" Button)
+	- On the right hand coloumn add labels to your issue to categorise it meaningfully
+	- Add it to board_1 **To Do** initially.
+
+- Creating a branch:
+	- In the NER-PM folder on your computer open a terminal window
+	- Type `git branch [name of branch]`
+	- Branches should follow this format: `i[issue #]-[contributor name]`. (e.g. `i18-james`)
+	- They type `git push` to push your branch to the repo. If this throws an error `fatal: The current branch i12-rahul has no upstream branch` type `git push --set-upstream origin [name of branch]` and then `git push`
+	- If you have been working on your branch for a while and the main repo has changed, type `git pull --rebase` to sync those changes to your branch
+	- If you close the terminal window and reopen it, you will be put back onto the master branch by default. This will give errors when trying to push so use `git switch [name of branch]` to switch back to your branch
+
+- Good coding practices:
+	- Add meaningful comments in the JSDoc style. [See line #10 of this code as an example](https://github.com/Northeastern-Electric-Racing/NER-PM-Dashboard/blob/master/Code_CovidFabForm.js)
```

Comment:
```suggestion
	- Add meaningful comments in the JSDoc style. [See line #10 of this code as an example](https://github.com/Northeastern-Electric-Racing/NER-PM-Dashboard/blob/master/server-code/CovidFabForm.js)
```

Reply:
For this should I keep the link or insert an image of the JSDoc from that file here so that people do not need to click into another file to look at JSDoc examples?

Reply:
Let's keep it as a link